docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,398 @@
+documents, fields, or clusters). To help you get up and running, there are 
+built-in roles. 
+
+Go to the *Management / Security / Roles* page to see them:
+
+[role="screenshot"]
+image::security/images/management-roles.jpg["Role management screenshot in Kibana"]
+
+Select a role to see more information about its privileges. For example, if you 
+select the `kibana_user` role, you will see that it grants `manage`, `read`, 
+`index`, and `delete` privileges on the `.kibana*` indices. To learn more about 
+these privileges, see <<privileges-list-indices>>. 
+
+Let's assign the `kibana_user` role to your user. Go back to the 
+*Management / Security / Users* page and select your user. Add the `kibana_user` 
+role and save the change. For example:
+
+[role="screenshot"]
+image::security/images/assign-role.jpg["Assigning a role to a user in Kibana"]
+
+This user now has the minimum privileges required to use {kib}. 
+
+If you completed all of the steps in <<get-started-elastic-stack>>, you should 
+have {metricbeat} data stored in {es}. Let's create two roles that grant 
+different levels of access to that data. 
+
+Go to the *Management / Security / Roles* page and click *Create role*. 
+
+Create a `metricbeat_reader` role that has `read` and `view_index_metadata` 
+privileges on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-reader-role.jpg["Creating a role in Kibana"]
+
+Create a `metricbeat_writer` role that has `manage_index_templates` and `monitor` 
+cluster privileges, as well as `write`, `delete`, and `create_index` privileges
+on the `metricbeat-*` indices:
+
+[role="screenshot"]
+image::security/images/create-writer-role.jpg["Creating another role in Kibana"]
+
+Now go back to the *Management / Security / Users* page and assign these roles 
+to the appropriate users. Assign the `metricbeat_reader` role to your personal 
+user.  Assign the `metricbeat_writer` role to the `logstash_internal` user.
+
+The list of users should now contain all of the built-in users as well as the 
+two you created. It should also show the appropriate roles for your users:
+
+[role="screenshot"]
+image::security/images/management-users.jpg["User management screenshot in Kibana"]
+
+If you want to learn more about authorization and roles, see <<authorization>>.
+
+[float]
+[[gs-logstash-security]]
+=== Add user information in {ls}
+
+In order for {ls} to send data successfully to {es}, you must configure its 
+authentication credentials in the {ls} configuration file.  For example, update 
+the `demo-metrics-pipeline.conf` file in your {ls} directory:
+
+[source,ruby]
+----
+input {
+  beats {
+    port => 5044
+  }
+}
+
+filter {
+  if [system][process] {
+    if [system][process][cmdline] {
+      grok {
+        match => {
+          "[system][process][cmdline]" => "^%{PATH:[system][process][cmdline_path]}"
+        }
+        remove_field => "[system][process][cmdline]"
+      }
+    }
+  }
+}
+
```

Comment:
If possible, I suggest having a shared file that contains the input and filter parts shown here. I worry that we'll update the example in one topic and forget to update it in another. Part of me wonders if it's even necessary to show the full config.

Reply:
Good point. I'll omit everything but the elasticsearch section